packages/web-app-files/src/store/actions.js

```diff
@@ -157,18 +164,30 @@ export default {
     if (space) {
       const promises = []
       const spaceShares = []
+      const userRoles = [
```

Comment:
Same here, can be collected dynamically by iterating over the `space.spaceRoles` entries.